MastermindLogic/src/main/java/nl/fhict/s3/mastermindlogic/entity/Row.java

```diff
@@ -36,4 +36,14 @@ public void setCode(EPinColour[] code) {
     public void setClues(EClueColour[] clues) {
         this.clues = clues;
     }
+
+    public boolean isEmpty() {
+
+        for(EPinColour pin : code) {
```

Comment:
Deze logica klopt niet, als de eerste pin null is geeft ie al true terug.

Reply:
Owja en misschien moeten we hier binnenkort maar even 1/2 tests voor schrijven

Reply:
ja dan is er iets empty dus dat klopt wel de naam zou nog anders mogen, maar de logica klopt.

Reply:
hasEmpty ipv isEmpty dan?